framework/src/org/checkerframework/framework/util/ContractsUtils.java

```diff
@@ -221,19 +225,96 @@ public int hashCode() {
         return result;
     }
 
-    /** Returns the annotation mirror as specified by the "qualifier" value in metaAnno. */
-    private AnnotationMirror getAnnotationMirrorOfQualifier(AnnotationMirror metaAnno) {
+    /**
+     * Returns the annotation mirror as specified by the "qualifier" value in {@code qualifierAnno}.
```

Comment:
Is this a helper function that should only be called in the two places it is currently called?  If so, I would document that.